### DISCUSSION TITLE
Mass-Redshift correlation?

### DISCUSSION THREAD

**Discussion by mikehelland** [a-cosmology-group/acg#221] [Cosmology Models]
In #213 I made a chart of metallicity over redshift and lookback time. I started working on one for mass measurements. If you know of any that should be included, please let me know.

![image](https://github.com/a-cosmology-group/acg/assets/19311107/dbea373a-1f74-455b-aecb-e0c8eca19642)

*Updated, May 28, added 1M galaxies from Zou, also added Caputi and Straatman.*

References and links here:

https://mikehelland.github.io/hubbles-law/other/stellarmass.htm

----

### Update: Jun-11-2024:

Here's a mass over redshift for galaxy clusters from "The M2C Galaxy Cluster Database":

https://www.galaxyclusterdb.eu/m2c/

And here's mass over time:

![M2C](https://github.com/a-cosmology-group/acg/assets/19311107/3bb77a39-0669-4395-87e2-02d81e8025f5)

**Comment by HanDeBruijn** [comment]
According to the Variable Mass Theory there does of course exist a _Mass-Redshift correlation_:

$$
1+z = \frac{m_0}{m} = e^{H_0(-t)}
$$

with $z=$ redshift, $m=$ mass, $H_0$ = Hubble parameter, $t=$ atomic time / "expanded time", previously called _measured time_ by you. 
But the picture shown above is consistent with so-called Minkowski time $\tau$, though there does not exist a physical clock running with it. The accompanying formula is

$$
-\tau = \frac{1}{H_0}\frac{z}{1+z}
$$

leading to a _Lookback Time_ of $1/H_0$ for $z=\infty$. I am affraid that other timescales are in better agreement with observation.

**Reply by mikehelland** [reply]
I don't know if this is related at all to your theory, but I saw this today:

https://arxiv.org/abs/2405.11869

**DESI constraints on varying electron mass model and axion-like early dark energy**

**Comment by LifeIsStrange** [comment]
Would be nice to add a line that shows the trend because visually it isn't rigorous.

**Comment by LifeIsStrange** [comment]
A subtopic about the mass redshift relation is the mass distribution/redshift relation
E.g what percentage of the observed mass is distributed around the galactic nucleus/"black hole" and around the disk and galactic bar.

Quote:
In the local Universe, the ratio between the mass of a central black hole and the stellar mass of its host galaxy is ∼0.1%. Recently, JWST discovered numerous galaxies at z>4 that seem to deviate from the local relation, with black holes overmassive by 10−100 times.

This recent study seems to find no "significant" evolution of central black hole size with redshift which is very surprising regarding latest JWST findings
https://arxiv.org/abs/2406.02779
Note that it is unclear to me whether they shows that the mean central black hole mass ratio in high Z jwst galaxies is normal or not.
From my limited understanding they test "intrinsic scatter" which I interpret as a test of galaxy "genealogy" meaning, in my neophyte interpretation, that they shows (with uncertainties) that the early galaxies with massive black holes would be a distinct population and that the low Z galaxies would be therefore not their descendants. 
IF this interpretation is correct then JWST stills finds evolution of central black hole mass over time but this would be distinct populations?

Another key question is the redshift evolution parameterized over the types of astrophysical objects e.g the ratio of spiral galaxies vs other types, blazar, etc. IIRC a major issue for static universes is the total extinction of quasars in the low redshift universe (assuming no intrinsic redshift) 
Note that this is related to the Tully fisher relation

**Reply by HanDeBruijn** [reply]
>Note that this is related to the Tully fisher relation

There is an simple derivation for the [Tully-Fisher relation](http://www.alternatievewiskunde.nl/cosmology/Hoyle.htm#TF) within (our version of) the [Variable Mass Theory](https://www.youtube.com/watch?v=T7jVviVBzKs).

**Reply by ExpEarth** [reply]
@LifeIsStrange `This recent study seems to find no "significant" evolution of central black hole size with redshift which is very surprising regarding latest JWST findings
https://arxiv.org/abs/2406.02779`

The study is just 3 pages.  It seems to be flailing a bit.  In the abstract they write:
`Using data from the local Universe (z < 0.055), low
(0.4 ≤ z ≤ 0.9), intermediate (0.9 ≤ z ≤ 4), and high (z > 4) redshift, we conclude that there is no
statistically significant redshift evolution of the intrinsic scatter.`

But in their conclusions, they write:
`Hence, current data is insufficient to conclude that there is a significant redshift evolution of the intrinsic scatter.`

**Comment by LifeIsStrange** [comment]
digression but spiral structures appears to evolve with redshift
https://arxiv.org/abs/2501.11670

**Comment by LifeIsStrange** [comment]
record low metallicity https://arxiv.org/abs/2501.11099
